src/Bus.ts

```diff
@@ -63,7 +59,7 @@ export class Bus {
 
         return returnValue;
       },
-      { Entries: [], FailedEntryCount: 0 } as {
+      { Entries: [], FailedEntryCount: entryTooBigCount } as {
```

Comment:
Je ne sais pas si c'est mieux de throw une erreur ou de retourner le résultat avec le nombre d'erreurs comme j'ai fais là... De notre côté, on ne voulait pas throw une erreur pour permettre au process de continuer.

Dans tous les cas, je pense que c'est important de ne pas throw une erreur **avant d'envoyer sur EventBridge** pour permettre aux events qui ne posent pas de problème, d'être effectivement bien envoyés.

Reply:
Aussi, peut-être que dans certains use cases, on ne veut rien envoyer si un event est trop gros. À ce moment-là, il faudrait peut-être avoir une option pour choisir ?

Reply:
You were right removing the `throw` instruction if input event is too large.
We should stick to EventBridge current behavior : this only thrown error is in case of Internal Server Error. Otherwise, even if some entries were not successfully published, the execution carries on. However, it is crucial that failed entries appear in the returning array with the source of the issue added to the object (https://docs.aws.amazon.com/eventbridge/latest/APIReference/API_PutEvents.html#API_PutEvents_ResponseElements). You can use expecting error format to add those entries in the return statement - https://docs.aws.amazon.com/eventbridge/latest/APIReference/API_PutEventsResultEntry.html